content/docs/user-guide/packaging/troubleshooting.md

```diff
@@ -24,9 +24,12 @@ The following are techniques you can use to help you debug any issues that you m
 In Visual Studio, you can compile with optimizations disabled and debug symbols enabled. This lets Visual Studio's debugging tools produce more helpful information that can help you debug.
 
 Make the following modifications to the `Configurations_msvc.cmake` file in the `<engine>\cmake\Platform\Common\MSVC` directory. In `ly_append_configurations_options`, under `COMPILATION _RELEASE`:
- - Change `/Od` to `/Ox`.
+ - Change `/Ox` to `/Od`.
  - Add `/Zi`.
 
+Make the following modifications under `LINK_NON_STATIC_RELEASE`:
+ - Add `/DEBUG`.
```

Comment:
Instead of having the user make manual modifications to `Configurations_<compiler>.cmake` files, why not make a CMake Variable that sets these compiler and linker options.
Such as `O3DE_ADD_DEBUG_SYMBOLS_TO_RELEASE` boolean cache variable?

Reply:
Related PR https://github.com/o3de/o3de/pull/16147